apps/heft/src/plugins/TypeScriptPlugin/TypeScriptPlugin.ts

```diff
@@ -76,17 +77,23 @@ export interface ISharedTypeScriptConfiguration {
   emitFolderNameForTests?: string;
 
   /**
-   * Set this to change the maximum number of file handles that will be opened concurrently for writing.
-   * The default is 50.
+   * Describes the way files should be statically coped from src to TS output folders.
```

Comment:
We should spell out `TS`.

```suggestion
   * Configures additional file types that should be copied into the TypeScript compiler's output folders, for example
   * so that these files can be resolved by import statements.
```